extras/parse_build_log.py

```diff
@@ -412,10 +424,10 @@ def makebuild_summary(log_filename, msg_make_section_start,
 
     if in_make_section:
         # 'make' was not completed.
-        return False, None, None, None, None
+        return False, None, None, None, None, None
     else:
         # There is no 'make' section at all.
-        return None, None, None, None, None
+        return None, None, None, None, None, None
```

Comment:
Thanks for catching this. Maybe you could add a comment explaining what the return values mean? That could potentially prevent such oversights later on.

Reply:
Done!